solutions/swb-app/src/projectRoutes.ts

```diff
@@ -3,20 +3,44 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { ProjectService } from '@aws/workbench-core-accounts';
-import CreateProjectRequest from '@aws/workbench-core-accounts/lib/models/createProjectRequest';
+import { ProjectService, CreateProjectRequest, ListProjectsRequest } from '@aws/workbench-core-accounts';
+import Boom from '@hapi/boom';
 import { Request, Response, Router } from 'express';
 import { validate } from 'jsonschema';
 import { wrapAsync } from './errorHandlers';
 import CreateProjectSchema from './schemas/projects/createProjectSchema';
+import ListProjectsSchema from './schemas/projects/listProjectsSchema';
 import { processValidatorResult } from './validatorHelper';
 
 export function setUpProjectRoutes(router: Router, projectService: ProjectService): void {
-  // Get projects
+  // List projects
   router.get(
     '/projects',
     wrapAsync(async (req: Request, res: Response) => {
-      const projects = await projectService.listProjects();
+      let projects = {};
+      // validate if query params present
+      if (req.query.pageSize || req.query.paginationToken) {
```

Comment:
nit: Invert if for readability
```
if (!req.query.pageSize && !req.query.paginationToken) {
  projects = await projectService.listProjects(res.locals.user);
  res.send(projects);
  return;
}
// rest code
```

Reply:
I'm going to include the user as part of the request per @aws-tyler 's recommendation so this check won't necessary